documentation/API-design-guidelines.md

```diff
@@ -1116,17 +1101,73 @@ When IpAddr is used in a payload the property objectType MUST be present to indi
 }
 ```
 
-### 11.6 OAuth Definition
+### 11.6 Security definition
+
+Finally, this part describes the security is applied to the API. 
```

Comment:
This section describes and aligns the security specific definitions within the API specification, such as scopes and the security schemes.

Reply:
Updated